packages/catalog-model/src/schema/kinds/Template.v1beta2.schema.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "TemplateV1beta1",
+  "description": "A Template describes a scaffolding task for use with the Scaffolder. It describes the required parameters as well as a series of steps that will be taken to execute the scaffolding task.",
+  "examples": [
+    {
+      "apiVersion": "backstage.io/v1beta1",
+      "kind": "Template",
+      "metadata": {
+        "name": "react-ssr-template",
+        "title": "React SSR Template",
+        "description": "Next.js application skeleton for creating isomorphic web applications.",
+        "tags": ["recommended", "react"]
+      },
+      "spec": {
+        "owner": "artist-relations-team",
+        "type": "website",
+        "parameters": {
+          "required": ["name", "description"],
+          "properties": {
+            "name": {
+              "title": "Name",
+              "type": "string",
+              "description": "Unique name of the component"
+            },
+            "description": {
+              "title": "Description",
+              "type": "string",
+              "description": "Description of the component"
+            }
+          }
+        },
+        "steps": [
+          {
+            "id": "fetch",
+            "name": "Fetch",
+            "action": "fetch:plain",
+            "parameters": {
+              "url": "./template"
+            }
+          },
+          {
+            "id": "publish",
+            "name": "Publish to GitHub",
+            "action": "publish:github",
+            "parameters": {
+              "repoUrl": "{{ parameters.repoUrl }}"
+            }
+          }
+        ],
+        "output": {
+          "catalogInfoUrl": "{{ steps.publish.output.catalogInfoUrl }}"
```

Comment:
This seems like something that will be used to allow the user to go to the page for the generated component. Will there be some required outputs that each template need to provide?

Really liking the direction this is taking 👍

Reply:
There will be some known outputs that the scaffolder frontend plugin knows how to display, but none of them are required

Reply:
Makes sense